src/get-started/java-binding.md

```diff
@@ -577,8 +625,8 @@ Currently Java Binding includes the following built-in services:
 ## Services Activation
 
 No services are enabled on the node by default. To enable services,
-define them in the `ejb_app_services.toml` configuration file.
-This file is required for a running node. `ejb_app_services.toml`
+define them in the `services.toml` configuration file.
```

Comment:
I didn't see any PRs for changing the corresponding section in `exonum-doc`. Just bothering to do not forget about that.